### DISCUSSION TITLE
Mute game on unfocused

### DISCUSSION THREAD

**Discussion by itsfprime** [ppy/osu#31264] [Ideas] [answered]
Adding an option to mute the game while unfocused would be pretty nice, I tab out when I run out of stamina and usually watch something online. Instead of having to go through the _**oh so tedious**_ act of scrolling down on the main menu, muting when I tab out would certainly be convenient!

**Comment by Joehuu** [comment, marked as answer]
You can change this to 0% to achieve what you want.

![Screenshot 2024-12-24 at 4 14 12 PM](https://github.com/user-attachments/assets/052962d6-f29c-4103-86cb-b67cb9e03e82)
